octree:
  resolution: 0.2
  lazy_evaluation: false
sensor_model:
  hit: 0.8
  miss: 0.4
  min: 0.03
  max: 0.97
  max_range: 30
  mismatch_penalty: 5
  empty_coeff: 0.000001
  occupied_coeff: 1
  free_coeff: 1
  algorithm_number: 4
filter:
  min_distance: 2
  max_distance: 40
ground_filter:
  enable: false
  iterations: 200
  distance_threshold: 0.05
  eps_angle: 0.15
  plane_distance: 0.04
map:
  # In units of meters, not cells
  # width of grid = width / resolution
  length: 200
  width: 200
  start_x: 100
  start_y: 100
  encoding: "CV_8UC1"
  log_odds_default: 0
node:
  lidar_topic: "/scan/pointcloud"
  transform_max_wait_time: 0.5
  debug: true
map_floor_threshold: 0.5
